application/frontend/controllers/MfaController.php

```diff
@@ -282,4 +307,48 @@ public function actionUpdate(int $id)
 
         return $mfa;
     }
+
+
+    /**
+     * @param int $mfaId
+     * @param int $webauthnId
+     * @return MfaWebauthn
+     * @throws BadRequestHttpException
+     * @throws NotFoundHttpException
+     * @throws ServerErrorHttpException
+     * @throws \Throwable
+     * @throws \yii\db\StaleObjectException
+     */
+    public function actionUpdateWebauthn(int $mfaId, int $webauthnId)
+    {
+        // check authorization
+        $this->getAuthorizedMfa($mfaId);
+
+        $label = \Yii::$app->request->getBodyParam('label');
+        if (empty($label)) {
+            throw new BadRequestHttpException('"label" must have a value in calls to update-webauthn', 1660233368);
+        }
```

Comment:
I think this "a label is required" check probably belongs in the MfaWebauthn model's `rules()` function, rather than here in the controller.

Reply:
Thanks for adding that to `rules()`. Remember to remove it from here in the Controller, too.